compiler/src/iree/compiler/Codegen/Dialect/Codegen/Utils/Utils.cpp

```diff
@@ -36,18 +36,7 @@ bool operator!=(const TileSwizzle &lhs, const TileSwizzle &rhs) {
 
 llvm::raw_ostream &operator<<(llvm::raw_ostream &os,
                               TileSwizzle::Dim::Kind kind) {
-  switch (kind) {
-  case TileSwizzle::Dim::Kind::Internal:
-    return os << "Internal";
-  case TileSwizzle::Dim::Kind::CrossThread:
-    return os << "CrossThread";
-  case TileSwizzle::Dim::Kind::CrossIntrinsic:
-    return os << "CrossIntrinsic";
-  default:
-    // Required by GCC.
```

Comment:
This line is being removed, but FYI in case a similar comment exists in the other place (convertSwizzleKindToString) or anywhere else: this comment incorrectly suggests that this `default:` only exists to please GCC. In fact, without this `default:`, this function reaches its end without a `return` statement, which isundefined behavior, if `kind` is not one of the `case` values. See https://github.com/llvm/llvm-project/issues/115345.

Reply:
I see, we do not have such comment in the other utility, so I think it's good. Good to know, thanks!

https://github.com/iree-org/iree/blob/2e2c109e8d58c8964ad7a3ba26d6addbfefbc1c4/compiler/src/iree/compiler/Codegen/Dialect/Codegen/Utils/Utils.cpp#L91-L103